### ISSUE TITLE
data: view count

### ISSUE THREAD

**Issue by tomholford** [tadad/blog#5]
%blog operator can optionally show a per-path view count in the footer of each page

**Comment by tadad** [comment]
yes - need to move the serving logic from a generator to the `+on-poke` arm, and then any GET can modify state. I have to do this for the upcoming `%theme` PR anyways so adding view count after that should be easy
